Gemfile

```diff
@@ -5,6 +5,10 @@ gemspec
 
 gem 'grape-swagger', github: 'ruby-grape/grape-swagger'
 
+unless RUBY_PLATFORM == 'java'
```

Comment:
RuboCop would complain. I'd make sure Gemfile is part of .rubocop.yml and `rubocop -a` this one.